src/extension.ts

```diff
@@ -30,6 +31,8 @@ export async function activate(context: vscode.ExtensionContext) {
         registerCommandWithTelemetry('aks.selectSubscriptions', selectSubscriptions);
         registerCommandWithTelemetry('aks.detectorDiagnostics', detectorDiagnostics);
         registerCommandWithTelemetry('aks.periscope', periscope);
+        registerCommand('azure-deploy.ConfigureCICDPipeline', configurePipeline),
```

Comment:
💡 Idea: if you are keen to make these commands be part of telemetry, use `registerCommandWithTelemetry` Also, If I have time, I can help you with this work if you need hand, but don't entirely know the scope yet so you can find me with internal alias. Thanks heaps.